test/integration/warm-pool/test_fns.go

```diff
@@ -0,0 +1,102 @@
+package warm_pool
+
+import (
+	"fmt"
+	"github.com/aws/amazon-vpc-cni-k8s/test/framework"
+	"github.com/aws/amazon-vpc-cni-k8s/test/framework/resources/k8s/manifest"
+	"github.com/aws/amazon-vpc-cni-k8s/test/framework/utils"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	v1 "k8s.io/api/apps/v1"
+	"strconv"
+)
+
+// Scales the cluster from minPods up to maxPods and back down to minPods
+func QuickScale(f *framework.Framework, deploymentSpec *manifest.DeploymentBuilder) (*v1.Deployment, error) {
+	// create deployment
+	deployment := deploymentSpec.Build()
+
+	_, err := f.K8sResourceManagers.
+		DeploymentManager().
+		CreateAndWaitTillDeploymentIsReady(deployment, utils.DefaultDeploymentReadyTimeout*5)
+	Expect(err).ToNot(HaveOccurred())
+
```

Comment:
We can skip this part and jump right into scaling up

Reply:
<img width="1434" alt="Screenshot 2023-09-15 at 9 28 23 AM" src="https://github.com/aws/amazon-vpc-cni-k8s/assets/67074258/db2c6b27-4a2e-40e0-ab31-8038e2141463">
If I'm understanding this correctly, you mean to skip the CreateAndWait?

Reply:
Skip that and the `deploymentSpec.Build`, as `changeReplicas` should be the only one to build the deployment

Reply:
Okay, so you want changeReplicas to create the deployment and then update the deployment as needed? I think I'm just getting thrown up incorporating the two different functions into changeReplicas.